### ISSUE TITLE
First example in "Introduction to Widgets" is difficult to see

### ISSUE THREAD

**Issue by aaronshenhao** [flutter/website#11923]
![Image](https://github.com/user-attachments/assets/7da59b02-cf3c-4c3b-aba3-ad23e475257e)

The issue occurs in both Firefox and Microsoft Edge. The text is imperceptible against the black background. Link: https://docs.flutter.dev/ui
